tests/scene/test_code_edit.h

```diff
@@ -3445,6 +3445,98 @@ TEST_CASE("[SceneTree][CodeEdit] completion") {
 		}
 	}
 
+	SUBCASE("[CodeEdit] autocomplete suggestion order") {
+		/* favorise less fragmented suggestion */
```

Comment:
```suggestion
		/* Favorize less fragmented suggestion. */
```
Sentence case and en_US spelling.

Likewise for others.